src/hw2/hw2.java

```diff
@@ -0,0 +1,161 @@
+        System.out.print("a = ");
+        int a = scanner.nextInt();
+        System.out.print("b = ");
+        int b = scanner.nextInt();
+        System.out.print("r = ");
+        int r = scanner.nextInt();
+
+        if (a < r * 2 & b < r * 2) {
+            System.out.println("True");
+        } else if (a >= r * 2 & b >= r * 2) {
+            System.out.println("False");
+        }
+
+
+        //конструкция выбора switch
+        Scanner scanner1 = new Scanner(System.in);
+        System.out.print("Enter n = ");
+        int n = scanner1.nextInt();
+
+        if (n > 10) {
+            System.out.println("False");
+        } else if (n <= 10) {
+            switch (n) {
+                case 1:
+                    System.out.print("Monday");
+                    break;
+                case 2:
+                    System.out.print("Tuesday");
+                    break;
+                case 3:
+                    System.out.print("Wednesday");
+                    break;
+                case 4:
+                    System.out.print("Thurday");
+                    break;
+                case 5:
+                    System.out.print("Friday");
+                    break;
+                case 6:
+                    System.out.print("Saturday");
+                    break;
+                case 7:
+                    System.out.print("Sunday");
+                    break;
+                default:
+                    System.out.print("Not a week day");
+            }
+        }
+
+        // циклы
+        for(int i = 1; i < 5; i++) {
+            System.out.println("****");
+        }
+
+        // массивы
+
+        // В векторе, состоящем из n вещественных элементов, вычислить: сумму отрицательных элементов вектора
+        // и произведение элементов вектора, расположенных между максимальным и минимальным элементами.
+
+        Scanner scanner2 = new Scanner(System.in);
+        System.out.println("Enter v = ");
+        int v = scanner2.nextInt();
+
+        int[] vector = new int[v];
+        for(int i=0; i < vector.length; i++){
+            vector[i] = (int)(Math.random() * v)-v;
+        }
+        int max = vector[0];
+        int min = vector[0];
+        for(int i = 0; i < vector.length; i++) {
+            int sum = 0;
+            sum = sum + (vector[i] <0);
+            if (min > vector[i])
+                min = vector[i];
+            if (max < vector[i])
+                max = vector[i];
+            else vector[i] *= vector[i];
+            System.out.println(sum);
+            System.out.println(max);
+            System.out.println(min);
+            System.out.println(vector[i] + " ");
+        }
+
+
+        // Создать и заполнить массив случайными числами, отсортировать массив по возрастанию,
+        // используя метод сортировки пузырьком.
+        int[] arr = new int[15];
+        for(int i = 0; i < arr.length; i++){
+            arr[i] = (int)(Math.random());
+            if( arr[i] > arr[i+1] ) {
```

Comment:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 15 out of bounds for length 15
	at Tmp.main(Tmp.java:10)